.gitignore

```diff
@@ -9,3 +9,6 @@ __pycache__/
 scripts/india_udise/udise_school_dropout_rate/data/*.json
 scripts/india_udise/common/data/3061_*.json
 scripts/india_udise/common/data/1003_*.json
+scripts/us_epa/superfund/site_and_funding_status/data/output
```

Comment:
you could also collapse these to:
scripts/us_epa/superfund/**/output

or move this into a .gitignore under scripts/us_epa/superfund

Reply:
Thank you for the suggestion, I have collapsed the gitignore file.